product_barcode_constraint_per_company/__manifest__.py

```diff
@@ -6,13 +6,13 @@
     "name": "Product Barcode Constraint per Company",
     "version": "14.0.1.0.1",
     "category": "Product",
-    "summary": "Change the product barcode constraint"
-    ", allowing the same barcode for differents companies",
+    "summary": """Change the product barcode constraint,
+                  allowing the same barcode for differents companies""",
```

Comment:
Don't change this. It was correct. Now, it isn't, as the spaces are included.

Reply:
Thanks for review!)
I'll fix it